src/PrestaShopBundle/Controller/Admin/Improve/Design/ThemeController.php

```diff
@@ -567,4 +555,43 @@ private function handleAdaptThemeToRTLLanguagesException(ThemeException $e)
 
         return $this->getFallbackErrorMessage($type, $e->getCode());
     }
+
+    /**
+     * Gets exception or exception and its code error mapping.
+     *
+     * @param DomainException $exception
+     *
+     * @return array
+     */
+    private function getLogoUploadErrorMessages(DomainException $exception)
+    {
+        $availableLogoFormatsImploded = implode(', .', ShopLogoSettings::AVAILABLE_LOGO_IMAGE_EXTENSIONS);
+        $availableIconFormat = ShopLogoSettings::AVAILABLE_ICON_IMAGE_EXTENSION;
+
+        $logoImageFormatError = $this->trans(
+            'Image format not recognized, allowed format(s) is(are): .%s',
+            'Admin.Notifications.Error',
+            [$availableLogoFormatsImploded]
+        );
+
+        $iconFormatError = $this->trans(
+            'Image format not recognized, allowed format(s) is(are): .%s',
+            'Admin.Notifications.Error',
+            [$availableIconFormat]
+        );
+
+        return [
+            NotSupportedLogoImageExtensionException::class => $logoImageFormatError,
+            NotSupportedFaviconExtensionException::class => $iconFormatError,
+            FileUploadException::class => [
+                UPLOAD_ERR_INI_SIZE => $this->trans(
+                    'File too large (limit of %s bytes).',
```

Comment:
Hi @LouiseBonnard  - does the message looks okey here? `File too large (limit of %s bytes).`. Previously it was `Image is too large (%1$d kB). Maximum allowed: %2$d kB` which in may opinion is way better but it is buggy since I can't get original file size then the error appears. The only suitable message I found is `File too large (limit of %s bytes).` but I always confused when I see number in bytes :D I would prefer Mb 

Do you have any suggestions of an existing error messages or should we implement something new here?

Reply:
It looks okay if the file size cannot be retrieved at this point. :-)